install/on_mac_os.md

```diff
@@ -17,27 +17,27 @@ Alternative there are `.tar.gz` and `.pkg` files in each [release](https://githu
 
 When a new Crystal version is released you can upgrade your system using:
 
-<div class="code_section">{% highlight bash %}
+```bash
```

Comment:
Oddly enough, this one is highlighted differently than the previous `` ```bash`` session. Not important, just a curiosity.

Reply:
I believe you mean the next one. The reason it didn't get properly highlighted is that the language was `txt` which is not recognized as a format by the highlighter. Thus it results in a different HTML structure, just a `pre` with `code.language-txt`.
Using a recognized language format fixes this.